open_spiel/games/twixt/twixt_test.cc

```diff
@@ -0,0 +1,212 @@
+// Copyright 2019 DeepMind Technologies Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "open_spiel/spiel.h"
+#include "open_spiel/tests/basic_tests.h"
+#include "open_spiel/games/twixt/twixt.h"
+
+namespace open_spiel {
+namespace twixt {
+namespace {
+
+namespace testing = open_spiel::testing;
+
+void BasicTwixTTests() {
+  testing::LoadGameTest("twixt");
+  testing::NoChanceOutcomesTest(*LoadGame("twixt"));
+  testing::RandomSimTest(*LoadGame("twixt"), 100);
+}
+
+class TestException : public std::exception {
+ public:
+    std::string error_msg_ = "";
+    char * what() {
+        return &error_msg_[0];
+    }
+
+    explicit TestException(const std::string& error_msg) {
+      error_msg_ = error_msg;
+    }
+};
+
+void ErrorHandler(const std::string& error_msg) {
+  std::cerr << "Twixt Fatal Error: " << error_msg << std::endl << std::flush;
+  throw TestException(error_msg);
+}
+
+
+
+void ParameterTest() {
+  std::string game_name = "twixt";
+  open_spiel::GameParameters params;
+  std::shared_ptr<const open_spiel::Game> game;
+  // ok: ansi_color_output=true
+  params.insert({"ansi_color_output", open_spiel::GameParameter(true, false)});
+  game = open_spiel::LoadGame(game_name, params);
+  params.clear();
+
+  // ok: board_size=10
+  params.insert({"board_size", open_spiel::GameParameter(10, false)});
+  game = open_spiel::LoadGame(game_name, params);
+  params.clear();
+
+  // too big: board_size=30
+  params.insert({"board_size", open_spiel::GameParameter(30, false)});
+  try {
```

Comment:
Try/catch  and exceptions are not allowed by Google C++ style. Please take a look at the other games tests and mimic what they do. Thanks!

Reply:
removed all try/catch blocks